test/speech/index.js

```diff
@@ -0,0 +1,836 @@
+  });
+
+  after(function() {
+    mockery.deregisterAll();
+    mockery.disable();
+  });
+
+  beforeEach(function() {
+    requestOverride = null;
+
+    speech = new Speech({
+      projectId: PROJECT_ID
+    });
+  });
+
+  describe('instantiation', function() {
+    it('should normalize the arguments', function() {
+      var normalizeArguments = fakeUtil.normalizeArguments;
+      var normalizeArgumentsCalled = false;
+      var fakeOptions = { projectId: PROJECT_ID };
+      var fakeContext = {};
+
+      fakeUtil.normalizeArguments = function(context, options) {
+        normalizeArgumentsCalled = true;
+        assert.strictEqual(context, fakeContext);
+        assert.strictEqual(options, fakeOptions);
+        return options;
+      };
+
+      Speech.call(fakeContext, fakeOptions);
+      assert(normalizeArgumentsCalled);
+
+      fakeUtil.normalizeArguments = normalizeArguments;
+    });
+
+    it('should inherit from Service', function() {
+      assert(speech instanceof Service);
+
+      var calledWith = speech.calledWith_[0];
+
+      var baseUrl = 'speech.googleapis.com';
+      assert.strictEqual(calledWith.baseUrl, baseUrl);
+      assert.strictEqual(calledWith.projectIdRequired, false);
+      assert.deepEqual(calledWith.scopes, [
+        'https://www.googleapis.com/auth/cloud-platform'
+      ]);
+      assert(speech.protos.Speech);
+      assert(speech.protos.Operations);
+    });
+  });
+
+  describe('recognize', function() {
+    var REQ = {};
+
+    it('should make the correct API request', function(done) {
+      var request = REQ;
+
+      speech.request = function(protoOpts, reqOpts) {
+        assert.strictEqual(protoOpts.service, 'Speech');
+        assert.strictEqual(protoOpts.method, 'syncRecognize');
+        assert.strictEqual(reqOpts, REQ);
+
+        done();
+      };
+
+      speech.recognize(request, assert.ifError);
+    });
+
+    describe('error', function() {
+      var error = new Error('Error.');
+      var RES = {};
+
+      beforeEach(function() {
+        speech.request = function(protoOpts, reqOpts, callback) {
+          callback(error, RES);
+        };
+      });
+
+      it('should execute callback with error & API response', function(done) {
+        speech.recognize(REQ, function(err, response, apiResponse) {
+          assert.strictEqual(err, error);
+          assert.strictEqual(response, null);
+          assert.strictEqual(apiResponse, RES);
+          done();
+        });
+      });
+    });
+
+    describe('success', function() {
+      var RES = {
+        results: []
+      };
+
+      beforeEach(function() {
+        speech.request = function(protoOpts, reqOpts, callback) {
+          callback(null, RES);
+        };
+      });
+
+      it('should execute callback with results & API resp', function(done) {
+        speech.recognize(REQ, function(err, response, apiResponse) {
+          assert.ifError(err);
+
+          assert(response.results);
```

Comment:
Couple things here:

1. I think we want to test that the response is an instance of `Speech.SyncRecognizeResponse`
2. We want to verify that the results are returned from `Speech.SyncRecognizeResponse`